server/liveblog/blogs/blogs.py

```diff
@@ -194,7 +203,10 @@ def on_updated(self, updates, original):
         recipients = []
         for user in members:
```

Comment:
I just remembered why I let aside creating a method for this check-up since it would have been to complicated. In the first case where I used `isinstance (user, ObjectId)` I then needed to get the whole user document after the verification in order to be able to extract its email. In the second case, I only needed to do the simple check-up.